jsonrpc/src/content/results.json

```diff
@@ -220,8 +220,8 @@
       "$ref": "#/components/schemas/hexString"
     }
   },
-  "GossipResult": {
-    "name": "gossipResult",
+  "PutContentResult": {
+    "name": "PutContentResult",
     "description": "Returns the number of peers that the content was gossiped to",
```

Comment:
Seems like this should also return information about whether the content was added to the database, and possibly if the data was already in the database.

Reply:
Returning a boolean to indicate if the data was written to the database makes sense. This can be implemented in the same way as portal_*Store currently does. Indicating if the data was already in the database would require a second database lookup to check if the key/data exists before overwriting so I'd say we shouldn't do this for performance reasons.